front/components/actions/dust_app_run/DustAppRunActionDetails.tsx

```diff
@@ -6,8 +6,8 @@ import { useMemo } from "react";
 import { ActionDetailsWrapper } from "@app/components/actions/ActionDetailsWrapper";
```

Comment:
Better to review with hidden whitespaces ([link](https://github.com/dust-tt/dust/pull/6177/files?diff=split&w=1)).